### PR TITLE
Adds all the necessary svgs with settable height and width

### PR DESCRIPTION
**Pull request by BenjaminParment**
-> Adds all the svgs needed from Catherine's design
-> Has height, width and onClick (optional) passed as props
-> Displays all the icons in App component to test they work (will delete before merging)

### CODE REVIEW

File: src/resources/svgs/ErrorOutlineIcon.tsx [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import React from "react";
+import { SVGProps } from "./svgProps";
+
+export class ErrorOutlineICon extends React.Component<SVGProps> {
```
Comment (j-n-white) [inline src/resources/svgs/ErrorOutlineIcon.tsx]:
```suggestion
export class ErrorOutlineIcon extends React.Component<SVGProps> {
```

File: src/App.tsx [hunk 2]
```diff
@@ -1,20 +1,46 @@
...
+            <ErrorOutlineICon height={x} width={x} />
+            <ExpandIcon height={x} width={x} />
+            <InfoIcon height={x} width={x} />
+            <KeyboardReturnIcon height={x} width={x} />
+            <RemoveIcon height={x} width={x} />
+            <SaveIcon height={x} width={x} />
+            <SearchIcon height={x} width={x} />
+            <SeeMoreHorizontalIcon height={x} width={x} />
+            <SeeMoreVerticalIcon height={x} width={x} />
+            <SettingsAppIcon height={x} width={x} />
```
Comment (j-n-white) [inline src/App.tsx]: As per my other review this all wants to move to the sandbox

File: src/resources/svgs/svgProps.ts [hunk 3]
```diff
@@ -0,0 +1,5 @@
+export interface SVGProps {
+    height: number;
+    width: number;
+    onClick?: () => void;
```
Comment (j-n-white) [inline src/resources/svgs/svgProps.ts]: We shouldn't need this as we will want to wrap a button round the icon when they are clickable.

File: src/App.tsx (hunk 2 above)
Comment (BenjaminParment) [reply src/App.tsx]: will do the change once add-redux has been merged

**Pull request merged**